app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def wrapper(*args, **kwargs) -> None:
+        if args in cache_dict:
+            print("Getting from cache")
+            return cache_dict[args]
+        else:
```

Comment:
There is no need for an `else` statement since the code will not execute after `return` anyway. Remove `else`.